core/string/unicode_normalize_spec.rb

```diff
@@ -0,0 +1,45 @@
+ruby_version_is "2.2" do
+  describe "String#unicode_normalize" do
+    it "returns nfc normalized form of string" do
+      "a\u0300".unicode_normalize.should == "à"
```

Comment:
I think it's useful here to also have the assertion when explicitly passing :nfc (the default argument).